examples/evaluation/correctness.ts

```diff
@@ -0,0 +1,42 @@
+import {
```

Comment:
How about adding
1. README for the new `evaluation` folder
2. docs for using `CorrectnessEvaluator`
?